storage/rocksdb/CMakeLists.txt

```diff
@@ -82,14 +101,10 @@ ENDIF()
 
 SET(rocksdb_static_libs ${rocksdb_static_libs} ${ZLIB_LIBRARY} "-lrt")
 
-MYSQL_ADD_PLUGIN(rocksdb_se ${ROCKSDB_SOURCES} STORAGE_ENGINE DEFAULT STATIC_ONLY
+MYSQL_ADD_PLUGIN(rocksdb ${ROCKSDB_SOURCES} STORAGE_ENGINE DEFAULT MODULE_ONLY
   LINK_LIBRARIES ${rocksdb_static_libs}
 )
 
-IF(WITH_EMBEDDED_SERVER)
```

Comment:
Stolen from TokuDB to be somewhat consistent since we don't build nor support Percona Server as embedded and will not likely ever support the same with RocksDB